docs/self-hosted/install/install-on-kubernetes.md

```diff
@@ -4,45 +4,46 @@ url: /docs/self-hosted/latest/install/install-on-kubernetes/
 
-Gitpod is installed using [Helm](https://helm.sh).
-The source of the chart can be found [here](https://github.com/gitpod-io/gitpod/blob/master/chart/).
-Releases are published at https://charts.gitpod.io.
+For some cloud providers we offer [Terraform](https://www.terraform.io/) scripts packaged into an installer that set up the infrastructure around Gitpod following performance-relevant best practices and using the cloud provider's managed services: Kubernetes cluster, database, storage, image registry. We recommend using the installer if available for your cloud provider. Once the installer has created the cluster, it will pre-configure and install this Gitpod helm chart into it.
+* [Installer for Google Cloud Platform (GCP)](../install-on-gcp-script/).
+* [Installer for Amazon Web Services (AWS)](../install-on-aws-script/).
 
 ## Prerequisites
- 1. Kubernetes version 1.15 <= x <= 1.17 is supported
+ 
+ 1. A Kubernetes cluster in version 1.15 <= x <= 1.17. 
+ 
+ 2. lcoal `kubectl` with connection to your cluster
+ 
+ 3. local `helm` in version >= 3.
 
- > Note: Gitpod should work on small nodes out of the box (2vCPUs, 8GB RAM). For a solid experience we recommend at least 4vCPUs and 16GB RAM for workspaces nodes.
+ Gitpod should work on small Kubernetes nodes out of the box (2vCPUs, 8GB RAM). For a better experience we recommend at least 4vCPUs and 16GB RAM for workspaces nodes. For cost efficienty, we recommend to enable cluster-autoscaling.
 
 ## Installation
 
-To initiate the deployment run the following commands:
+To install Gitpod in your Kubernetes cluster follow these steps:
+
+Run the following commands in your local terminal:
 ```console
 helm repo add gitpod.io https://charts.gitpod.io
 
 helm install gitpod gitpod.io/gitpod
 ```
 
- > Review the deployment worked properly by running `kubectl get pods`. Eventually all pods should be up-and-running. In case they are not have a look the the [Troubleshooting Guide](../troubleshooting/)
- 
- 1. Configure [ingress into the cluster](../configure-ingress/).
-    It makes sense to create a new folder for all Gitpod specific custom configuration and files:
-    ```
-    mkdir gitpod && cd gitpod
-    ```
+Now run `kubectl get pods` and verify that all pods are in state `RUNNING`. If some are not, please see the [Troubleshooting Guide](../troubleshooting/).
+
+Configure [domain and https](../configure-ingress/).
+
  2. Go to [https://\<your-domain.com\>](https://\<your-domain.com\>) and follow the steps to complete the installation.
 
+
 ## Recommended Configuration
 
-Without further configuration the Helm chart installs a working Gitpod installation in a lot of scenarios.
-Yet, there are certain things you might want to review when installing Gitpod for long term use and/or a bigger audience:
+By default, the Helm chart installs sub-charts for a MySQL Database, Storage via MinIO and a Docker Registry. If you want to use other services instead, please change the following configuration:
```

Comment:
nit: I like the old text better in every respect except the intro "by default" :slightly_smiling_face: